examples/chart/teleport-cluster/templates/proxy/_config.common.tpl

```diff
@@ -0,0 +1,72 @@
+{{- define "teleport-cluster.proxy.config.common" -}}
+{{- $logLevel := (coalesce .Values.logLevel .Values.log.level "INFO") -}}
```

Comment:
This `coalesce` was a backwards compatibility workaround for when we switched from `logLevel` to `log.level`. I wonder whether it could be removed now to clean things up a bit, given that the use of `logLevel` was deprecated several major versions ago? Not a huge deal if we want to preserve absolute backwards compatibility though.

Reply:
I think I'll keep this for now and drop this in v13 to reduce the amount of breaking changes here.